### PR TITLE
Fosology playbook Authored

### PR DESCRIPTION
**Pull request by samirparhi-dev**
Fosology playbook Authored Please review

### CODE REVIEW

File: playbooks/besman-fossology_license_compliance-playbook.sh [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        docker run --name fossology -p 8081:80 --d fossology/fossology
+    fi
+
+}
+
+function __besman_execute {
+
+    # Define variables
+
+    FOSSOLOGY_HOST_URL_WITH_PORT_NO="$2"
```
Comment (asa1997) [inline playbooks/besman-fossology_license_compliance-playbook.sh:53]: Move the actual execution into a steps file.

File: playbooks/besman-fossology_license_compliance-playbook.sh [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+function __besman_cleanup {
+
+    echo -e "\n${BLUE}✅ Cleaning-UP and Reclaiming your Used space !${RESET}"
+    # Stop Fossology container
+    echo -e "\n${BLUE}Stopping Fossology container...${RESET}"
+
+    docker stop fossology
+
+    sleep 10
```
Comment (asa1997) [inline playbooks/besman-fossology_license_compliance-playbook.sh:131]: Why is this required?

File: playbooks/besman-fossology_license_compliance-playbook.sh [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    
+
+    if [ $? -eq 0 ]; then
```
Comment (asa1997) [inline playbooks/besman-fossology_license_compliance-playbook.sh:134]: This checks for sleep and not the docker stop command

**Review by asa1997 — CHANGES_REQUESTED** [review]
@samirparhi-dev Please look at the comments. Update the metadata json file

**Review by pramit-d — CHANGES_REQUESTED** [review]
@samirparhi-dev Please add a version for playbooks. The file name should contain the version number. Remove sync, codeql, and sonar-scanner playbook files from this PR as they are already added in another PR.
Use [sbom generator](https://github.com/Be-Secure/besecure-playbooks-store/blob/main/playbooks/besman-spdx-sbom-generator-0.0.1-playbook.sh) playbook as an example, and modify the file accordingly.

**Pull request closed without merging**